lib/gui/app/components/target-selector/target-selector-modal.tsx

```diff
@@ -96,22 +101,15 @@ const TargetsTable = styled(({ refFn, ...props }) => {
 		</div>
 	);
 })`
-	[data-display='table-head']
-		[data-display='table-row']
-		> [data-display='table-cell']:first-child {
-		padding-left: 15px;
-	}
-
-	[data-display='table-head']
-		[data-display='table-row']
-		> [data-display='table-cell'] {
+	[data-display='table-head'] [data-display='table-cell'] {
+		position: sticky;
+		top: 0;
+		background-color: rgb(248, 249, 253);
 		padding: 6px 8px;
 		color: #2a506f;
 	}
 
-	[data-display='table-body']
-		> [data-display='table-row']
-		> [data-display='table-cell']:first-child {
+	[data-display='table-cell']:first-child {
```

Comment:
This whole style can be refactored to:
```css
[data-display='table-head'] [data-display='table-cell'] {
		position: sticky;
		top: 0;
	}

	[data-display='table-cell']:first-child {
		padding-left: 15px;
	}

	&& [data-display='table-row']
		> [data-display='table-cell'] {
		padding: 6px 8px;
		color: #2a506f;
	}
```
so the last rule is valid for both table head and body and is more specific than rendition's rules (which in the `head` override the padding)